compressor/gzip/gzip.go

```diff
@@ -18,32 +18,126 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-// Package gzip provides a YARPC binding for gzip compression.
-package gzip
+// Package yarpcgzip provides a YARPC binding for gzip compression.
+package yarpcgzip
```

Comment:
this change looks unrelated...

Reply:
Thanks. The perils of dropping everything to fix a thing…